src/components/Circles/LargePink.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
```

Comment:
This code is difficult to understand. Even you, the **writer** will have difficulty understanding it if you open this code 2 weeks or a month from now.  These are design elements , not used anywhere else in design- so making them components is not required. 
All elements in this - `Largepink` `purple` `smallpink` They are used once- for background of "Join Madhyam" This all should be part of that component or in the same folder at the very least

Reply:
Moved the circles folder which was on the components folder to joinMadhyam components.